iac/prd/us-east-1/region.hcl

```diff
@@ -0,0 +1,4 @@
+locals {
+  aws_region = "us-east-1"
+  vpc_id     = "vpc-058d97f9b63161ea7"
```

Comment:
Should the VPC ID be hard-coded?

Reply:
In terms of it being sensitive data? It will be a constant vpc, because we are only deploying it once. I don't think resource IDs are considered sensitive in any way. If you disagree I'll remove it.

Reply:
Eh, it's fine.